buildenv

```diff
@@ -51,13 +51,20 @@ zopen_check_results()
 zopen_install()
 {
   set -e
-  mkdir -p $ZOPEN_INSTALL_DIR
-  rm -rf $PWD/zotsampleport
-  cp -r $PWD/ $ZOPEN_INSTALL_DIR/
+  mkdir -p "$ZOPEN_INSTALL_DIR"
+  rm -rf "$PWD/zotsampleport"
+  cp -r "$PWD/" "$ZOPEN_INSTALL_DIR/"
   set +e
 }
 
 zopen_append_to_env()
 {
-  echo "export SSL_CERT_FILE=\"\$PWD/cacert.pem\""
+  echo "export SSL_CERT_FILE=\"\$PWD/../cacert.pem\""
+}
+
+
+zopen_append_to_setup() {
+cat <<EOF
+rm -rf "\$PWD/meta/.git"* "\$PWD/meta/.editorconfig" "\$PWD/meta/.env" && cp -r "\$PWD/meta/." "\$PWD/" && rm -rf "\$PWD/meta" && cp "\$PWD/cacert.pem" "\$PWD/../cacert.pem"
```

Comment:
Should we keep the .git file? I kinda prefer it because then I can do a git pull to get the latest.

Reply:
but this is in the 'installed' (prod) location. Wouldn't you checkout meta as a git repo to do development on it directly? I found this a bit awkward myself I have to say because metaport isn't smart enough to look for a local dev instance and instead always pulls from github...

Reply:
Ok, that makes sense. We could run `zopen upgrade meta` the achieve similar